src/components/accessories/patientTherapy/therapyForm/TherapyForm.tsx

```diff
@@ -29,11 +31,27 @@ const TherapyForm: FC<TherapyProps> = ({
   resetFormCallback,
 }) => {
   const validationSchema = object({
-    // TODO
+    medicalId: string().required("This field is required"),
+    startDate: string().required("This field is required"),
+    endDate: string().required("This field is required"),
```

Comment:
Here you should use the i18n function in order to retrieve the translated string from the dictionary.

Reply:
OK